docker-compose.build.yaml

```diff
@@ -29,6 +29,13 @@ services:
       context: airbyte-workers
       labels:
         io.airbyte.git-revision: ${GIT_REVISION}
+    #connector-builder:
```

Comment:
(commented out)
this adds the connector-builder server to the docker-compose deploy